sklearn/feature_selection/_from_model.py

```diff
@@ -60,6 +60,15 @@ def _calculate_threshold(estimator, importances, threshold):
     return threshold
 
 
+def _estimator_has(attr):
+    """Check if `self.estimator_` (if available) or `self.estimator` has `attr`"""
```

Comment:
```suggestion
    ```suggestion
    """Check if the provided estimator has the method implemented.
    
    We check if the method is available in the fitted estimator first
    (i.e. `self.estimator_`). If not available, we check the estimator passed
    at the initialization (i.e. `self.estimator`).
    """
```
```